src/components/auth/login/SignUpScreen.js

```diff
@@ -138,7 +139,8 @@ const getStylesFromProps = ({ theme }) => {
       // minHeight: getDesignRelativeHeight(256),
     },
```

Comment:
```suggestion
    },
    signupIcon: {
      viewBox: '0 0 11 22'
    },
```

Reply:
This is not a style prop, it's an svg attribute we pass directly as a prop